extensions/default/src/Panels/Toolbox/ToolboxUI.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
This belongs in the UI section, as it is a generic tool that can be used - it really isn't specific to an mode or extension.